demos/kernelmethod/demo_client.py

```diff
@@ -65,5 +83,45 @@
     elif mode == 'inference':
         client.load_data(data_path=data_path, feature_names=cfg.get('Data', 'features').split(','))
         client.model = client.load_model(model_path=cfg.get('Model', 'model_path'))
+    else:
+        raise ValueError("Invalid mode!")
+    return client
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    # load config file
+    parser.add_argument('-C', '--config_path', type=str, required=True, help='path of the configuration file')
+    parser.add_argument('-F', '--flag_network', type=str, required=False, default="F", help='flag to use new network api')
+    
+    args = parser.parse_args()
+
+    cfg = ConfigParser()
+    cfg.read(args.config_path)
+
+    client = load_client(cfg, args)
+    
+    if ("flag_network" not in args) or (args.flag_network == "F"):
+        # old api framework
```

Comment:
Hmm, I understand what is going on here, but I suggest putting a more clear description as comments.
Meanwhile it is clear but in the future it's hard to rename it as "new new API" right?
Either we put a todo or mark it with versions so we can backtrack if needed.